### ISSUE TITLE
ELMCIP accessibility testing with tota11y

### ISSUE THREAD

**Issue by steinmb** [elmcip/elmcip#352]
Try to make sure the site is not breaking too many accessibility rules. Basic stuff like headers, 
contrast, font sizes, image properties.

* Follow [WCAG 2.0](https://www.w3.org/TR/WCAG20/) recommendations.
* Test with [Tota11y](https://github.com/Khan/tota11y) toolkit

## Scope

* [ ] Verify site header menu links etc.
* [ ] Verify site footer region
* [ ] Verify node forms in general

**Comment by steinmb** [comment]
## Front page

![elmcip___electronic_literature_as_a_model_of_creativity_and_innovation_in_practice](https://user-images.githubusercontent.com/683617/31882018-c2fbdc74-b7e5-11e7-987d-ea8be22e512a.png)

> To give your document a proper structure for assistive technologies, it is important to lay out your headings beginning with an `<h1>`. We found an `<h2>` instead.

**Comment by steinmb** [comment]
## Front page contrast ratio

![elmcip___electronic_literature_as_a_model_of_creativity_and_innovation_in_practice](https://user-images.githubusercontent.com/683617/31882245-686aa49c-b7e6-11e7-8e47-b7591a801add.png)


>The color combination #ffffff/#bebdae has a contrast ratio of 1.90, which is not sufficient. At this size, you will need a ratio of at least 3.

**Comment by steinmb** [comment]
Need more time though some improvement is in this branch:

2027fb5 (HEAD -> issue_352, origin/issue_352) Issue 352 Set page background color to white.
32887ed Issue 352 Refactor and improve hover states to menu item
a3eb69b Issue 352 Refactor and change header element color

**Comment by steinmb** [comment]
Top header region blow out of page wrapper and create a full width area.

<img width="1275" alt="screen shot 2018-04-21 at 12 52 39" src="https://user-images.githubusercontent.com/683617/39083844-646a7c36-4563-11e8-8b40-b749407d64ac.png">


commit 2625841a5e793e7aebf05ec83cf136c86974b622 (HEAD -> issue_352, origin/issue_352)
Author: Stein Magne Bjorklund <steinmb@smbjorklund.com>
Date:   Sat Apr 21 12:52:03 2018 +0100

    Issue 352 Introduce full width functionality. Used in header

**Comment by steinmb** [comment]
Looking at how the forms looked on a older projector, is was pretty clear to me that the contrast used for example in vertical forms is too low.

**Comment by steinmb** [comment]
commit baa4073a1271de3137c728cb8ec7447014e6e233 (HEAD -> master, origin/master, origin/HEAD)
Author: Stein Magne Bjorklund <steinmb@smbjorklund.com>
Date:   Tue Jan 19 15:30:40 2021 +0100

    Issue 352 Social networks accessibility links lack title

    Confusing to screen readers hyperlinks with no contextual
    information.

**Comment by steinmb** [comment]
Solved by new theme improvments - https://elmcip.net/story/elmcip-and-cell-v138-released
